src/components/Test.vue

```diff
@@ -128,6 +136,12 @@ export default {
     async loadTest(testId) {
       try {
         this.test = await firebaseService.loadTest(testId);
+        console.log('expirationDate:', this.test.expirationDate);
+        const expirationDate = this.test.expirationDate || new Date().getTime();
```

Comment:
I think we should still let the user take the test in the last day. That means we should default to another value.

Reply:
> I think we should still let the user take the test in the last day. That means we should default to another value.

The expiration date should be mandatory when tests are generated, to make sure we know the final datetime when users can send solutions. The current time is a failover in case expiration date is missing entirely.